docs/rules/valid-api.md

```diff
@@ -1,6 +1,13 @@
 # Validate `api` decorator usage (valid-api)
 
-This rule checks whether every `api` decorator is valid.
+The following restriction applies the `api` decorator.
+
+-   The `api` decorator can only be applied to class fields and class methods.
+-   Properties and method should be unique per class.
```

Comment:
```suggestion
-   Fields and methods should be unique per class.
```

Reply:
@pmdartus it seems like only the first rule applies to `@api`, the rest seem to be about field and method names. Am I misunderstanding? 

Also, the first rule uses "fields" so I changed the subsequent instances of "properties" to "fields," is that OK?

Reply:
All the rules apply to field and fields and APIs annotated with the `@api` decorator. Maybe it should be a little more explicit. For instance, the next line should be understood as _Fields and methods [anotated with `@api` decorator] can't start with `on`. The `on` prefix is reserved to bind event handlers_.

Do you think it worth adding the implicit _anotated with `@api` decorator_ on all the lines?